packages/xo-web/src/xo-app/user/index.js

```diff
@@ -282,6 +289,85 @@ const SshKeys = addSubscriptions({
   )
 })
 
+// ===================================================================
+const dateFormat = d3.timeFormat('%Y-%m-%d %H:%M')
```

Comment:
Can we use [`ShortDate`](https://github.com/vatesfr/xen-orchestra/blob/web_auth_token_crud/packages/xo-web/src/common/utils.js#L519-L521) or something similar?

Reply:
IMO, in a "`SortedTable` it's more friendly to have a numeric date. Personally, I feel more comfortable seeing and sorting numeric dates.

Reply:
I wasn't clear enough: I meant doing the same thing as what we did for `ShortDate`, ie creating a tiny helper in `common/utils.js` that uses `FormattedDate` and that we can reuse if needed, instead of using a different library for this specific case. What do you think?